docs/src/remote-wallet/ledger.md

```diff
@@ -12,6 +12,27 @@ secure transaction signing.
 
 ## Install the Solana App on Ledger Nano S
 
+The Solana Ledger app is not yet available on Ledger Live. Until it is, you can
+install a development version of the app from the command-line. Note that
+because the app is not installed via Ledger Live, you will see the message,
+"This app is not genuine" each time you open the app. Once the app is available
+on Ledger Live, you can reinstall the app from there, and the message will no
+longer be displayed.
+
+1. Connect and unlock your Ledger Nano S
+2. Download and run the Solana Ledger app installer:
+   ```text
+   curl -sSLf https://github.com/solana-labs/ledger-app-solana/releases/download/v0.1.1/install.sh | sh
+   ```
+3. When prompted, approve the "unsafe" manager on your device by pressing the right button.
```

Comment:
Now that I'm reading it in multiple sentences, this wording ("pressing the right button") seems a little awkward; it seems to suggest one right button push is enough to signify approval. Technically, you have to press the right button (or the left... which is what I do, as it's shorter) until you get to the approval screen and then press both buttons. Not sure how specific we ought to be about device physical operation, though...

Reply:
Fixed in 99cc943. No longer dictating how to approve.  That was more micromanaging than anything else in the instructions.